Libs/Mesh/MeshUtils.h

```diff
@@ -33,6 +33,9 @@ class MeshUtils
 
   /// determine the reference mesh
   static int findReferenceMesh(std::vector<Mesh> &meshes);
+
+  /// return array of average normals for each point in these meshes
+  static Mesh computeMeanNormals(std::vector<Mesh> &meshes);
```

Comment:
See comment in cpp: pass a const reference to a vector of std::reference_wrapper(const Mesh) like the bounding box function in this class.

Reply:
Done